docs/src/main/sphinx/release/release-0.69.md

```diff
@@ -8,7 +8,7 @@ The following config properties must be removed from the
 - `presto-metastore.db.filename`
 
 Additionally, the `datasources` property is now deprecated
-and should also be removed (see [Datasource Configuration]).
+and should also be removed (see [Datasource Configuration](datasource-configuration)).
```

Comment:
How about just say 

> see Datasource Configuration below

without a link. Or see if we can do a local anchor using `#`. Not sure we want a global reference to an ancient release note page.

Reply:
I changed the name of the link to reduce the impact on the overall namespace. I also tried other options but sphinx either doesnt let it pass the build or it just renders it wrongly so its not a link at all. I think we can go with the current approach.